homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -26,11 +26,12 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class MaxCubeShutter(BinarySensorEntity):
     """Representation of a MAX! Cube Binary Sensor device."""
 
-    def __init__(self, handler, name, rf_address):
+    def __init__(self, handler, name, device):
         """Initialize MAX! Cube BinarySensorEntity."""
         self._name = name
         self._sensor_type = DEVICE_CLASS_WINDOW
-        self._rf_address = rf_address
+        self._serial = f"{DOMAIN}.{device.serial}"
```

Comment:
Is the serial number only unique in the context of a domain for a MAX!Cube?

Reply:
It is a 10 character string of alphanumerical characters unique for all devices handled by maxcube component. I prefixed it with "maxcube." to make it globally unique. Is it needed?

Reply:
I can change it to just device.serial if the unique_ids does not need to be globally unique but only unique within a component.

Reply:
The ID only needs to be unique within the domain: https://developers.home-assistant.io/docs/entity_registry_index/#unique-id

More uniqueness presumably doesn't hurt, but I think you can remove DOMAIN here.

Reply:
Good. Should I create a new merge request or can I force-update this one?